app/workers/reactions/update_relevant_scores_worker.rb

```diff
@@ -9,7 +9,11 @@ def perform(reaction_id)
       return unless reaction&.reactable
 
       reaction.reactable.touch_by_reaction if reaction.reactable.respond_to?(:touch_by_reaction)
-      reaction.reactable.sync_reactions_count if rand(6) == 1 && reaction.reactable.respond_to?(:sync_reactions_count)
+      if reaction.reactable.respond_to?(:sync_reactions_count)
+        ThrottledCall.perform(:sync_reactions_count, throttle_for: 15.minutes) do
```

Comment:
Should we throw a test in for this?

Reply:
My plan has been foiled... 😉 

![image](https://user-images.githubusercontent.com/47985/109120355-6d644200-7778-11eb-9269-72b3243d3cc9.png)